SConstruct

```diff
@@ -74,3 +74,4 @@ envCython["LIBS"] = python_libs
 Export('envCython')
 
 SConscript(['opendbc/can/SConscript'])
+SConscript(['opendbc/dbc/generator/SConscript'])
```

Comment:
these should go in a wrapper SConscript so openpilot only has to call one. see how the other submodules do it